src/display/annotation_layer.js

```diff
@@ -1565,6 +1568,7 @@ class AnnotationLayer {
         svgFactory: new DOMSVGFactory(),
         annotationStorage:
           parameters.annotationStorage || new AnnotationStorage(),
+        enableScripting: parameters.enableScripting,
```

Comment:
You also need to update the `AnnotationElementParameters`-typedef at the top of this file, hence please add the following there:
```javascript
 * @property {boolean} [enableScripting] - Enable embedded script execution.
 *   The default value is `false`.
```